src/phoenix/server/api/queries.py

```diff
@@ -69,8 +71,8 @@
 from phoenix.server.api.types.Span import Span, to_gql_span
 from phoenix.server.api.types.SystemApiKey import SystemApiKey
 from phoenix.server.api.types.Trace import Trace
-from phoenix.server.api.types.User import User
-from phoenix.server.api.types.UserApiKey import UserApiKey
+from phoenix.server.api.types.User import User, from_orm_user
+from phoenix.server.api.types.UserApiKey import UserApiKey, from_orm_api_key
```

Comment:
```suggestion
from phoenix.server.api.types.User import User, to_gql_user
from phoenix.server.api.types.UserApiKey import UserApiKey, to_gql_api_key
```
A bit reversed in naming no?

Reply:
good call